blivet/gpt.py

```diff
@@ -183,6 +183,18 @@
 }
 
 
+_gpt_discoverable_partitions = False
```

Comment:
Please use `blivet.flags.Flags` here instead. We already have multiple flags for controlling various default behaviours, so it make more sense having this there too. Anaconda is also setting these flags, so it will make it easier for the installer too.

Reply:
Ah yes, that's a much better place for this.